caiman/cluster.py

```diff
@@ -353,6 +353,10 @@ def setup_cluster(backend: str = 'multiprocessing',
             'multiprocessing' [alias 'local'], 'ipyparallel', and 'SLURM'
             ipyparallel and SLURM backends try to restart if cluster running.
             backend='multiprocessing' raises an exception if a cluster is running.
+        n_processes: int
+            Set number of processes to use. If None, is automatically calculated 
+        single_thread: bool
+            If True, do not use cluster
```

Comment:
Is cluster use the right way to phrase this?

Reply:
I'm open to suggestions, but the module is called `cluster.py` and that word is pretty ubiquitous

Reply:
How about "If True, dview is set to None, so a single core is used"?

Also tempted to change doc on `dview` to something more general like "multicore processing object" as discussed in #1070